### PR TITLE
AP_Periph:Update README.md to remove ADSB on AP_Periph

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: Tools/AP_Periph/README.md [hunk 1]
```diff
@@ -29,7 +29,6 @@ UAVCAN sensor types. Support is included for:
  - Barometers (SPI or I2C)
  - Airspeed sensors (I2C)
  - Rangefinders (UART or I2C)
- - ADSB (Ping ADSB receiver on UART)
```
Comment (peterbarker) [inline Tools/AP_Periph/README.md:32]: This should be `uAvionix Ping ADSB`... 

The line can be here, but the current version is misleading.

I'll be working on getting proper support for the library in very soon.

**Review by peterbarker — CHANGES_REQUESTED** [review]

File: Tools/AP_Periph/README.md (hunk 1 above)
Comment (Hwurzburg) [reply Tools/AP_Periph/README.md:32]: then what was this about? https://discuss.ardupilot.org/t/adsb-using-ap-periph-not-connecting/105634/4

Comment (rmackay9) [reply Tools/AP_Periph/README.md:32]: ... at the risk of being pushy, maybe give @peterbarker a few days and if it remains a WIP then we remove it for now and put it back when his PR is ready... or just remove it now and put it back later.

Comment (peterbarker) [reply Tools/AP_Periph/README.md:32]: > then what was this about? https://discuss.ardupilot.org/t/adsb-using-ap-periph-not-connecting/105634/4

I've responded on that Discuss post.

**Pull request closed without merging**